### PR TITLE
Merge Badge Component

### PR DESCRIPTION
**Pull request by Hamdrive**
Created 2 types of badges
1. badge for icons (top right and left positions)(single and multiple digits)
2. badges for avatars (dot badge and border badge), used avatar component as well to generate example

image

### CODE REVIEW

File: components/badge/badge.css [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+  border-radius: var(--radius-rounded);
+  position: absolute;
+  top: 0;
+  right: 0;
+}
+
+.badge-dot-md {
+  width: var(--size-7);
+  height: var(--size-7);
+  border-radius: var(--radius-rounded);
```
Comment (shivampandey0) [inline components/badge/badge.css]: these 4 properties are repeating in the above 3 classes.

File: components/badge/badge.css [hunk 2]
```diff
@@ -0,0 +1,114 @@
+/* parent selector */
+
+.badge {
+  position: relative;
+}
+
+/* Icon badges */
+
+.icon {
```
Comment (shivampandey0) [inline components/badge/badge.css]: there could be any kinds of icons on a site, maybe icons.css a separate file?

File: components/badge/badge.css [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+  height: var(--size-12);
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  padding: var(--size-3);
+  font-size: var(--size-9);
+  font-weight: var(--weight-medium);
+}
+
+.top-right {
```
Comment (shivampandey0) [inline components/badge/badge.css]: could go in utilities

File: components/badge/badge.css [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+  top: 0;
+  right: 0;
+  transform: translate(50%, -50%);
+}
+
+.top-left {
```
Comment (shivampandey0) [inline components/badge/badge.css]: could go in utilities

**Review by shivampandey0 — COMMENTED** [review]
Added some suggestions.

File: components/badge/badge.css [hunk 5]
```diff
@@ -0,0 +1,114 @@
...
+}
+
+.unit {
+  position: absolute;
+  border-radius: var(--radius-rounded);
+  min-width: var(--size-12);
+  height: var(--size-12);
+  display: flex;
+  justify-content: center;
+  align-items: center;
```
Comment (rdshinde) [inline components/badge/badge.css]: I think this set of properties to Center the  item can be separated.

**Review by rdshinde — COMMENTED** [review]
Looks good.

File: components/badge/badge.css (hunk 1 above)
Comment (Hamdrive) [reply components/badge/badge.css]: Noted, have created a new CSS class property to store these 4 lines and use them in the respective badges

File: components/badge/badge.css (hunk 2 above)
Comment (Hamdrive) [reply components/badge/badge.css]: Have renamed classes to be specific to badges

File: components/badge/badge.css (hunk 3 above)
Comment (Hamdrive) [reply components/badge/badge.css]: Done, created new utilities.css, will use it for its intended purpose

File: components/badge/badge.css (hunk 4 above)
Comment (Hamdrive) [reply components/badge/badge.css]: Done

File: components/badge/badge.css (hunk 5 above)
Comment (Hamdrive) [reply components/badge/badge.css]: I have added this to utilities.css, with a flexbox section. Thanks for recommendation

**Comment by Hamdrive** [comment]
Applied changes based on reviews, merging to dev

**Review by NtshVrm — APPROVED** [review]

**Pull request merged**